datalad/crawler/nodes/tests/test_misc.py

```diff
@@ -471,4 +473,11 @@ def node(d):
 def test_debug():
     yield _test_debug, "About to run"
     yield _test_debug, "Ran node .* which yielded 1 times", ('after',)
-    yield _test_debug, "Ran node .* which yielded 0 times", ('empty',)
\ No newline at end of file
+    yield _test_debug, "Ran node .* which yielded 0 times", ('empty',)
+
+
+def test_fix_url():
+    # TODO: actual test
+    print("Test")
```

Comment:
In case that `print` was meant to be a marker for you to spot your test in nose's output, you can run that test only via `nosetests -s -v datalad/crawler/nodes/tests/test_misc.py:test_fix_url`.
If the purpose was different - just ignore me. ;-)